lang/wasm_instance_cache.hh

```diff
@@ -52,11 +53,28 @@ public:
         uint64_t cache_blocks = 0;
     };
 
-    static stats& shard_stats() {
-        static thread_local stats _stats;
+private:
+    stats _stats;
+    seastar::metrics::metric_groups _metrics;
+
+public:
+    stats& shard_stats() {
         return _stats;
     }
 
+    void setup_metrics() {
```

Comment:
ah, nvm, I see it's done here; should be moved to the previous patch

Reply:
Ok